src/Components/TeamCard.jsx

```diff
@@ -1,59 +1,52 @@
 import {
   Avatar,
   Card,
-  makeStyles,
   IconButton,
   CardMedia,
   List,
   ListItem,
   ListItemText,
 } from "@material-ui/core";
 import { LinkedIn } from "@material-ui/icons";
-
-const useStyles = makeStyles({
-  card: {
-    borderRadius: 0,
-    width: 275,
-  },
-  name: {
-    fontWeight: "bold",
-  },
-  avatar: {
-    width: "100%",
-    height: 250,
-  },
-});
+import { useAboutStyles } from '../Styles/useAboutStyles';
 
 const TeamCard = (props) => {
-  const classes = useStyles();
+  const classes = useAboutStyles();
   const { firstName, lastName, role, linkedIn } = props.member;
 
   return (
-    <Card className={classes.card}>
+    <Card className={classes.card} tabIndex={0}>
       <CardMedia>
         <Avatar
           className={classes.avatar}
           variant="square"
           src={`https://cvp-team-photos.s3.us-west-2.amazonaws.com/${
```

Comment:
Some reason Madeline and Katie's pictures throw an error. 403 forbidden

Reply:
They did not have photos uploaded to the s3 bucket with the rest. I will do another round of collecting photos.